scrapy_poet/utils.py

```diff
@@ -14,3 +16,15 @@ def get_scrapy_data_path(createdir: bool = True, default_dir: str = ".scrapy") -
     if createdir:
         os.makedirs(path, exist_ok=True)
     return path
+
+
+def scrapy_response_to_http_response(response: Response):
+    """Convenience method to convert a ``scrapy.http.Response`` into a
+    ``web_poet.HttpResponse``.
+    """
+    return HttpResponse(
```

Comment:
This is probably a rare edge case, but it seems the conversion won't work properly for TextResponses where encoding is set explicitly, in `__init__` method. We're not passing this encoding, which means HttpResponse would try to guess the encoding again, and the result could be different.